### ISSUE TITLE
Rust Error on ParseTicks When Using isAlive and inventory

### ISSUE THREAD

**Issue by dbalders** [LaihoE/demoparser#55]
Error: When running the ParseTicks command, if you have both the `isAlive` and `inventory` attributes selected, it will throw the following error:

```
thread '<unnamed>' panicked at demoparser/src/parser/src/variants.rs:319:21:
Tried to push a Some(Bool(false)) into a StringVec([["m4a1_silencer", "smokegrenade"]]) column
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
```

Demo: Both overpass and inferno from this match: https://www.hltv.org/matches/2367592/complexity-vs-m80-thunderpick-world-championship-2023

Version: 0.11.3

Command: `let playerData = parseTicks(filePath, ["is_alive", "inventory"], [singleTick])`

**Comment by LaihoE** [comment]
should be fixed in 0.12.0

**Comment by dbalders** [comment]
Looks good, thank you very much!
